lib/from_honeybee/_openapi/honeybee_workflow.osw

```diff
@@ -0,0 +1,47 @@
+{
+  "seed_file": null,
+  "weather_file": null,
+  "measure_paths": [],
+  "file_paths": [],
+  "run_directory": null,
+  "steps": [
+    {
+      "measure_dir_name": "from_honeybee_simulation_parameter",
+      "arguments": {
+        "simulation_parameter_json": "multi_zone_model.json"
+      }
+    },{
+      "measure_dir_name": "from_honeybee_model",
+      "arguments": {
+        "model_json": "multi_zone_model.json"
+      }
+    },{
+      "name": "create_typical_building_from_model",
+      "measure_dir_name": "create_typical_building_from_model",
+      "arguments": {
+        "__SKIP__": true,
+        "template": "90.1-2004",
+        "add_constructions": false,
+        "add_space_type_loads": false,
+        "add_elevators": false,
+        "add_exterior_lights": false,
+        "add_exhaust": false,
+        "add_swh": false,
+        "add_refrigeration": false,
+        "remove_objects": false,
+        "system_type": "Inferred",
+        "add_hvac": true,
+        "use_upstream_args": false
+      }
+    },{
+      "measure_dir_name": "default_feature_reports",
+      "arguments": {
```

Comment:
I realize that this osw is mostly for demonstration purposes but I think we will `__SKIP__` this measure by default and only run it as a separate step when it is requested.